webapp/graphite/functions/params.py

```diff
@@ -21,6 +26,41 @@ class ParamTypes(object):
   setattr(ParamTypes, paramType, paramType)
 
 
+def validateBoolean(value):
+  return isinstance(value, bool)
+
+
+def validateFloat(value):
+  return isinstance(value, float)
```

Comment:
What happens if we pass an "integer" value to a parameter that expects a float? I suspect that the evaluator (since it doesn't interact with the param definition at all) will pass it in as an integer type and this will fail.